### PR TITLE
Use geopy to calculate the distance between two points(Duplicate of #5339)

### PR DESCRIPTION
**Pull request by Joshuathuo**
## Our **Peer Reviewer and Content Moderator teams do NOT provide any revisions services.** All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner. 

**ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.**

>As a rule of thumb - please be sure to only submit articles (pull requests) that are fully polished and ready to be published. Be sure to go through our resources documents for extra and 3rd party (vetted) resources to help improve overall technical writing.

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [ ] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

`This closes #issue-number`
This closes #5339
> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by WanjaMIKE** [comment]
Plagiarism check done. Please correct the highlighted issues.
[New Scan 8_50 AM Copyleaks Report.pdf](https://github.com/section-engineering-education/engineering-education/files/7761334/New.Scan.8_50.AM.Copyleaks.Report.pdf)

File: content/articles/calculating-the-distance-between-two-points-using-geopy/index.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+pip install geopy
+```
+
+Using Geopy, we can calculate or find the distance between two positions using the below methods:
+
+- Geodesic measure.
+- The great circle distance.
+- Herversine formula.
+
+### Geodesic measure
```
Comment (WanjaMIKE) [inline content/articles/calculating-the-distance-between-two-points-using-geopy/index.md:43]: In this section, you appear to be defining geodesic several times thus repeating yourself. Please make changes.

**Comment by Joshuathuo** [comment]
@WanjaMIKE I have corrected that

**Comment by WanjaMIKE** [comment]
After this review, your article appears quite short minus the code. You will need to add more content for more value.

**Comment by WanjaMIKE** [comment]
@Joshuathuo  If you need time, you can rework the PR outside the queue.

**Comment by Joshuathuo** [comment]
@WanjaMIKE you can reopen so that I can submit

**Comment by WanjaMIKE** [comment]
Reopened

**Comment by Joshuathuo** [comment]
@WanjaMIKE I have submitted

File: content/articles/calculating-the-distance-between-two-points-using-geopy/index.md [hunk 2]
```diff
@@ -24,6 +24,15 @@ We will explain how this process is possible using [geopy](https://geopy.readthe
...
 
+But before executing the command below, there is a series of steps that are followed so that the command is finally arrived at.
+1. Open your cmd and make sure it is running as an administrator.
+2. Make sure that you are operating from the root by performing the following.
+cd..
+
+3. cd.. This shows that you are already at the root.
+
+4. After that now you can perform the command below and geopy will be installed in your machine.
+
```
Comment (WanjaMIKE) [inline content/articles/calculating-the-distance-between-two-points-using-geopy/index.md:35]: The above addition seems insufficient and do not add value to the article.

**Comment by WanjaMIKE** [comment]
Your article is still less than 750 words minus the code. It seems you need more time. Please create a duplicate PR when ready. Thanks

**Comment by Joshuathuo** [comment]
@WanjaMIKE according to the topic itself most part of it is the code  which means the words will be lesser.. And may I ask is the code not included in the tally of the 750 words.

**Comment by WanjaMIKE** [comment]
No its not included in the word count

**Comment by Joshuathuo** [comment]
But on my side the words excluding the code are more than 750

**Comment by Joshuathuo** [comment]
@WanjaMIKE  The words from my side are more than 750 excluding the code

**Comment by WanjaMIKE** [comment]
They are actually about 600 words. You can paste the file in MS Word then remove all code and front matter. Please add more content then create a duplicate PR. Thanks

**Pull request closed without merging**